test/cmdlineTests/model_checker_targets_all/err

```diff
@@ -1,33 +1,3 @@
-Warning: CHC: Underflow (resulting value less than 0) happens here.
```

Comment:
These tests seem to be assuming that `default` is synonymous with "all". Maybe a new value that actually means "all" should be introduced? Or maybe just tests should be renamed to have `_default` in their dir names?

Reply:
Yea, maybe `default` is better

Reply:
Wait no, this test actually assumes that default is *not* all, that's why underflow is not reported anymore.

Reply:
Sorry, I wasn't completely clear that I was talking only about the name, not about the way the test was updated. It was originally meant to test all targets and it's no longer doing that. The updated test is ok for testing `default`, so just renaming it would be fine.

But this gave me an idea that maybe a value that actually means `all` should be introduced. Just a suggestion though - not sure if it would be useful in practice.

Reply:
Ah ok, makes sense

Reply:
Ah, the `all` is actually referring to the engines, but I'll rename to make it more clear